src/test/java/it/pagopa/pn/cucumber/steps/pa/AvanzamentoNotificheB2b.java

```diff
@@ -274,5 +276,43 @@ public void ilDestinatarioLeggeLaNotificaRicevuta(String recipient) {
     }
 
 
+    @Then("viene verificato che la chiave dell'attestazione opponibile {string} è {string}")
+    public void vieneVerificatoCheLaChiaveDellAttestazioneOpponibileÈ(String legalFactCategory, String key) {
```

Comment:
se possibile il metodo chiamiamolo in inglese così non abbiamo problemi con lettere accentate. Che dici?

Reply:
Giusto, per il momento ho corretto il nome di questo test, verranno aggiornati tutti i nomi dei meotdi.